en/docs/quick-starts/age-based-adaptive-auth.md

```diff
@@ -22,28 +16,27 @@ The instructions below guide you through specifying authentication steps based o
 
 ## Configure claims
 
-1.  Start the server and log in to the [management console](insertlink).
+1.  Start the server and log in to the Management Console (`https://<IS_HOST>:<PORT>/carbon`).
 
-2.  Click **List** under **Claims** and click
-    [http://wso2.org/claims](https://localhost:9443/carbon/identity-claim-mgt/list-local-claims.jsp).
+2.  Click **List** under **Claims** and click http://wso2.org/claims (`https://localhost:9443/carbon/identity-claim-mgt/list-local-claims.jsp`).
 
 3.  Click on the **Edit** corresponding to the **BirthDate** claim.
 
 4.  Select the **Supported By Default** checkbox to enable the birth
     date claim.  
 
-    ![enable dob claim](../assets/img/samples/enable-dob-claim.png)
+    ![enable dob claim](../../../assets/img/samples/enable-dob-claim.png)
 
 ----
 
 ## Add users
 
 1.  Create a user called "Alex" with login permission.
 
-    For instructions, see [Adding Users and Roles](insertlink).
+    For instructions, see [Adding Users](../../../guides/identity-lifecycles/admin-creation-workflow/).
```

Comment:
since Alex should have login permission, shouldn't we provide a link to adding roles?